main.go

```diff
@@ -719,5 +719,9 @@ func ToUTC(v interface{}) interface{} {
 	if tm, ok := v.(time.Time); ok {
 		return tm.UTC()
 	}
+	if tm, ok := v.(*time.Time); ok && tm != nil {
```

Comment:
REQ: How about creating tests for this function and also updating tests for [scope.go](https://github.com/housinganywhere/gorm/blob/ba9440b7ebf4533038c5aac5e256ee76019344e5/scope_test.go) to include this case?